src/platform_impl/windows/event_loop.rs

```diff
@@ -1513,31 +1524,31 @@ unsafe fn public_window_callback_inner<T: 'static>(
                 *SKIP_POINTER_FRAME_MESSAGES,
                 *GET_POINTER_DEVICE_RECTS,
             ) {
-                let pointer_id = LOWORD(wparam as DWORD) as UINT;
-                let mut entries_count = 0 as UINT;
-                let mut pointers_count = 0 as UINT;
-                if GetPointerFrameInfoHistory(
+                let pointer_id = (wparam & 0xffff) as u32; //LOWORD
+                let mut entries_count = 0_u32;
+                let mut pointers_count = 0_u32;
+                if !GetPointerFrameInfoHistory(
                     pointer_id,
-                    &mut entries_count as *mut _,
-                    &mut pointers_count as *mut _,
+                    &mut entries_count,
+                    &mut pointers_count,
                     std::ptr::null_mut(),
-                ) == 0
+                ) == true.into()
                 {
                     return 0;
                 }
 
                 let pointer_info_count = (entries_count * pointers_count) as usize;
                 let mut pointer_infos = Vec::with_capacity(pointer_info_count);
-                pointer_infos.set_len(pointer_info_count);
-                if GetPointerFrameInfoHistory(
+                if !GetPointerFrameInfoHistory(
                     pointer_id,
-                    &mut entries_count as *mut _,
-                    &mut pointers_count as *mut _,
+                    &mut entries_count,
+                    &mut pointers_count,
                     pointer_infos.as_mut_ptr(),
-                ) == 0
+                ) == true.into()
```

Comment:
Semantic change (negative comparison to `true` instead of positive comparison to `false`)